src/browserlib/extract-events.mjs

```diff
@@ -0,0 +1,278 @@
+      if (matchingInterfaces.length === 1) {
+	// only one such handler, we assume it's a match
+	return matchingInterfaces;
+      } else {
+	console.error("[reffy] Multiple event handler named " + eventEl.textContent + ", cannot associate reliably to an interface in " + spec.title);
+      }
+    }
+  }
+
+
+  let events = [];
+  // Look for event summary tables
+  // ignore DOM spec which uses a matching table format
+  // to map to legacy event types
+  let hasStructuredData = false;
+  if (spec.shortname !== "dom") {
+    document.querySelectorAll("table").forEach(table => {
+      const firstHeading = table.querySelector("thead tr th")?.textContent?.trim();
+      if (firstHeading && firstHeading.match(/^Event/) && firstHeading !== "Event handler") {
+	hasStructuredData = true;
+	// Useful e.g. for pointerevents
+	const bubblingInfoColumn = [...table.querySelectorAll("thead th")].findIndex(n => n.textContent.trim().match(/^bubbl/i));
+	const interfaceColumn = [...table.querySelectorAll("thead th")].findIndex(n => n.textContent.trim().match(/^interface/i));
+	table.querySelectorAll("tbody tr").forEach(tr => {
+	  const event = {};
+	  const eventEl = tr.querySelector("*:first-child");
+	  const annotations = eventEl.querySelectorAll("aside, .mdn-anno");
+	  annotations.forEach(n => n.remove());
+
+	  let el = eventEl.querySelector("dfn,a");
+	  if (el && el.tagName === "A" && el.getAttribute("href").startsWith("https:")) {
+	    // we skip when we hit a link pointing to an external spec
+	    // (this is needed since the HTML spec table includes
+	    // links to pointer events)
+	    return;
+	  }
+	  if (!el) {
+	    el = eventEl.querySelector("code");
+	  }
+	  if (el) {
+	    if (el.tagName === "DFN" && el.id) {
+	      event.href = href(el);
+	    } else if (el.tagName === "A") {
+	      event.href = href(document.getElementById(el.getAttribute("href").slice(1)));
+	    }
+	    event.src = { format: "summary table", href: href(el.closest('*[id]')) };
+	    event.type = eventEl.textContent.trim();
+	    event.targets = fromEventElementToTargetInterfaces(eventEl.querySelector("dfn,a[href^='#']"));
+	    if (bubblingInfoColumn >= 0) {
+	      event.bubbles = tr.querySelector(`td:nth-child(${bubblingInfoColumn + 1})`)?.textContent?.trim() === "Yes";
+	    }
+	    if (interfaceColumn >= 0) {
+	      event.interface = tr.querySelector(`td:nth-child(${interfaceColumn + 1}) a`)?.textContent ?? tr.querySelector(`td:nth-child(${interfaceColumn + 1}) code`)?.textContent;
+	    }
+	    events.push(event);
+	  }
+	});
+      } else if (table.className === "event-definition") {
+	hasStructuredData = true;
+	// Format used e.g. in uievents
+	const eventName = table.querySelector("tbody tr:first-child td:nth-child(2)")?.textContent.trim();
+	let iface = table.querySelector("tbody tr:nth-child(2) td:nth-child(2)")?.textContent.trim();
+	let bubbles = table.querySelector("tbody tr:nth-child(4) td:nth-child(2)")?.textContent.trim() === "Yes";
```

Comment:
Going to need another `?` before call to `trim()`, as in `?.textContent?.trim()`